CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [#51](https://github.com/rubocop-hq/rubocop-rails/issues/51): Add allowed receiver class names option for `Rails/DynamicFindBy`. ([@tejasbubane][])
 * [#211](https://github.com/rubocop-hq/rubocop-rails/issues/211): Add autocorrect to `Rails/RakeEnvironment` cop. ([@tejasbubane][])
 * [#242](https://github.com/rubocop-hq/rubocop-rails/pull/242): Add `Rails/ContentTag` cop. ([@tabuchi0919][])
+* [#xxx](): Add `Rails/BeforeDestroy` cop. ([@ecbrodie][])
```

Comment:
Please use the new changelog format https://github.com/rubocop/rubocop-rails/commit/3cae05d8cf1a94e76496a42bf38e033f941690f6#diff-84b0e0ad32ba865c4f40649f0e502782883e4f0d90767d50e13d684bf9674c0d